src/containers/About.jsx

```diff
@@ -1,23 +1,23 @@
-import React, { Component } from 'react';
-import '../assets/container/about.css'
-import Card from 'react-bootstrap/Card'
+import React, {Component} from "react";
+import "../assets/container/about.css";
+import Card from "react-bootstrap/Card";
 // Images temporary ideally these will be hosted online, and we won't import from assets
-import ayush from '../assets/images/ayush.jpg';
-import yg from '../assets/images/yg.jpg';
-import katherine from '../assets/images/katherine.jpg';
-import spencer from '../assets/images/spencer.jpg';
-import jenny from '../assets/images/jenny.jpg';
-import roxane from '../assets/images/roxane.jpg';
-import elisa from '../assets/images/spencer.jpg';
-import mayank from '../assets/images/spencer.jpg';
-import bilal from '../assets/images/bilal.jpg';
+import ayush from "../assets/images/ayush.jpg";
+import yg from "../assets/images/yg.jpg";
+import katherine from "../assets/images/katherine.jpg";
+import spencer from "../assets/images/spencer.jpg";
+import jenny from "../assets/images/jenny.jpg";
+import roxane from "../assets/images/roxane.jpg";
+import elisa from "../assets/images/spencer.jpg";
+import mayank from "../assets/images/spencer.jpg";
+import bilal from "../assets/images/bilal.jpg";
```

Comment:
unrelated to this PR, but you can put these files in your public folder (instead of assets) and add a link to them using
`url.resolve(process.env.PUBLIC_URL, "/path/within/public")` that has the same effect

Reply:
What's the benefit of putting it in the public folder?